images/macos/macos-13-Readme.md

```diff
@@ -67,36 +73,33 @@
 - zstd 1.5.5
 
 ### Tools
-- AWS CLI 2.13.29
-- AWS SAM CLI 1.99.0
+- AWS CLI 2.13.36
+- AWS SAM CLI 1.102.0
 - AWS Session Manager CLI 1.2.497.0
-- Azure CLI 2.53.1
+- Azure CLI 2.54.0
 - Azure CLI (azure-devops) 0.26.0
-- Bicep CLI 0.22.6
-- Cabal 3.6.2.0
+- Bicep CLI 0.23.1
 - Cmake 3.27.7
-- CodeQL Action Bundle 2.15.1
-- Fastlane 2.216.0
-- GHC 9.8.1
-- GHCup 0.1.19.4
-- Stack 2.13.1
-- SwiftFormat 0.52.8
+- CodeQL Action Bundle 2.15.2
+- Fastlane 2.217.0
+- Stack stack:
```

Comment:
@Alexey-Ayupov looks like a leftover in software report

Reply:
Will fix it